src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx

```diff
@@ -87,7 +73,7 @@ const ArtworkSidebarPartnerInfo: React.FC<ArtworkSidebarPartnerInfoProps> = ({
           )}
         </PartnerContainer>
 
-        {shouldRenderContactGalleryCTA && (
+        {!!partner.isInquireable && (
```

Comment:
**question**: should we hide `Contact Gallery` button in [ArtworkSidebarCommercialButtons](https://github.com/artsy/force/blob/main/src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx#L382-L395) in this case?

Reply:
Good question, first thing that comes to mind is that we don't need to do it, but looking into it now you might be right

Reply:
You are right, found one example of a sold artwork that with the new implementation has the contact gallery  button plus this button, so two contact gallery buttons 🤔 will bring it up with design

Reply:
[artwork link](https://www.artsy.net/artwork/casey-van-loon-nicos-blanket) for reference, will spawn a review app so we can share etc

Reply:
opened a thread and resolved it [here](https://artsy.slack.com/archives/C9SATFLUU/p1676296699409339), pushed the new changes

Reply:
great catch @dimatretyak